### ISSUE TITLE
DOAJ without full text

### ISSUE THREAD

**Issue by nemobis** [dissemin/doai#2]
http://dx.doi.org/10.3389/fncel.2015.00266 redirects to http://journal.frontiersin.org/article/10.3389/fncel.2015.00266/full which is open access; http://doai.io/10.3389/fncel.2015.00266 goes to https://doaj.org/article/ff0e27627f5147cba867a83563c75085 which only has the abstract and links to http://journal.frontiersin.org/article/10.3389/fncel.2015.00266/full for the full text.

I don't know in which cases DOAJ has the full text (I thought they never held it?!), but better check before linking them.

P.s.: I already sent you a few emails, sorry; I think the issue tracker is more tidy.

**Comment by wetneb** [comment]
You're right, maybe we should strip off DOAJ links. When they're present, that generally means the DOI is open.
